src/bam-fetcher-worker.js

```diff
@@ -952,7 +867,7 @@ const renderSegments = (
       Math.floor((maxPos - minPos) / maxCoverageSamples),
       1,
     );
-    const result = getCoverage(segmentList, coverageSamplingDistance);
+    const result = getCoverage(uid, segmentList, coverageSamplingDistance);
```

Comment:
What is `uid` here?

Reply:
Ah yes. Makes sense 👍  A comment would be helpful. You can see how poor my memory is and I wrote a bunch of the pileup code :-)